projects/Mallard/src/screens/settings/privacy-policy-screen.tsx

```diff
@@ -3,10 +3,8 @@ import { DefaultInfoTextWebview } from './default-info-text-webview'
e a reasonable fee if your request is clearly unfounded, repetitive or excessive.</p></li><li><p>Where you have provided us with consent to use your personal data, you can withdraw this at any time.</p></li><li><p>Where applicable, you may also have a right to receive a machine-readable copy of your personal data.</p></li><li><p>You also have the right to ask us to delete your personal data or restrict how it is used. There may be exceptions to the right to erasure for specific legal reasons which, if applicable, we will set out for you in response to your request. Where applicable, you have the right to object to processing of your personal data for certain purposes.</p></li><li><p>If you do not want us to use your personal data for marketing analysis, you can change your settings in the “Emails and marketing” tab of your Guardian account.</p></li></ul><p>If you want to make any of these requests, please contact dataprotection@theguardian.com.</p><p>We may need to request specific information from you to help us confirm your identity.</p><h2><strong>Your California privacy rights</strong></h2><p>Under California Civil Code Section 1798.83, if you live in California and your business relationship with us is mainly for personal, family or household purposes, you may ask us about the information we release to other organisations for their marketing purposes. To make such a request, please send an email to dataprotection@theguardian.com with “Request for California privacy information”in the subject line. You may make this type of request once every calendar year. We will email you a list of categories of personal data we revealed to other organisations for their marketing purposes in the last calendar year, along with their names and addresses. Not all personal data shared in this way is covered by Section 1798.83 of the California Civil Code.</p><h2><strong>Contact us for information about how we use your personal data</strong></h2><p>If you have any questions about how we use your personal data or if you have a concern about how your personal data is used, please contact the Data Protection Officer at Guardian News &amp; Media Limited, Kings Place, 90 York Way, London N1 9GU. Or, email dataprotection@theguardian.com.</p><p>Complaints will be dealt with by the Data Protection Officer, and will be responded to within 30 days.</p><p>If you are not satisfied with the way your concern has been handled, you can refer your complaint to the Information Commissioner’s Office.</p><p>If you have a question about anything else, please see our Contact us information under Settings -&gt; Help in this app.</p><p><strong>Changes to the Privacy Policy</strong></p><p>If we decide to change our privacy policy we will post the changes here. If the changes are significant, we may also choose to email all our registered users with the new details. If required by law, we will get your permission or give you the opportunity to opt out of any new uses of your data.</p>"`
```

Comment:
Looks like a syntax error at the end here? Extra quote?

Reply:
Updated.